README.md

```diff
@@ -30,27 +51,233 @@ npm install --save ipfsd-ctl
 
 IPFS daemons are already easy to start and stop, but this module is here to do it from JavaScript itself.
 
+### Local node
+
 ```js
 // Start a disposable node, and get access to the api
-// print the node id, and kill the temporary daemon
-
-// IPFS_PATH will point to /tmp/ipfs_***** and will be
-// cleaned up when the process exits.
+// print the node id, and stop the temporary daemon
 
-var ipfsd = require('ipfsd-ctl')
+const DaemonFactory = require('ipfsd-ctl')
+const df = DaemonFactory.create()
 
-ipfsd.disposableApi(function (err, ipfs) {
-  ipfs.id(function (err, id) {
+df.spawn(function (err, ipfsd) {
+  const ipfsCtl = ipfsd.ctl
+  const ipfsCtrl = ipfsd.ctrl
+  ipfsCtl.id(function (err, id) {
```

Comment:
Very confusing names.  

How about `api` for `ctl` and `daemon` for `ctrl`?

Or just return one object that is the `js-api` monkey patched with the `ctrl` methods?

Reply:
Agree, it's confusing.
Instead of monkey patching, perhaps the `.spawn` could return the ctrl object and callback with the ipfs node interface?

Reply:
To clarify, the two proposals here are

```
// Rename
const ipfsApi = ipfsd.api
const ipfsDaemon = ipfsd.daemon
```

And

```
Return the two things in two different places:
const ipfsDaemon = df.spawn(function (err, ipfsApi) {
  //...
})
```

I'm fine with the rename. I have mixed feelings with returning one thing and callback the other.

Reply:
I agree, I think the rename would be better.